test/parallel/test-console.js

```diff
@@ -76,17 +102,33 @@ assert.equal('foo bar\n', strings.shift());
 assert.equal('foo bar hop\n', strings.shift());
```

Comment:
This section is a bit of a mess in that it's not very readable and mixes the strings and errStrings checks up. This can likely be simplified by creating a secondary array with the expected strings in the right order then just doing a for-loop through each of strings and errStrings to check each one. The code would be much easier to read and there'd be much less of it :-)